package.json

```diff
@@ -26,6 +26,7 @@
     "@docusaurus/preset-classic": "^3.5.2",
     "@docusaurus/types": "^3.5.2",
     "@floating-ui/react-dom": "1.3.0",
+    "@fluentui/react": "^8.123.5",
```

Comment:
Why the new icon set? Should we remove the old and swap it out with this one so there are not two?